backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/materialDispo/forecast.feature

```diff
@@ -0,0 +1,92 @@
+      | Identifier | M_PriceList_Version_ID.Identifier | M_Product_ID.Identifier | PriceStd | C_UOM_ID.X12DE355 | C_TaxCategory_ID.InternalName |
+      | pp_1       | plv_1                             | p_1                     | 10.0     | PCE               | Normal                        |
+
+    And metasfresh contains PP_Product_BOM
+      | Identifier | M_Product_ID.Identifier | ValidFrom  | PP_Product_BOMVersions_ID.Identifier |
+      | bom_1      | p_1                     | 2021-04-01 | bomVersions_1                        |
+    And metasfresh contains PP_Product_BOMLines
+      | Identifier | PP_Product_BOM_ID.Identifier | M_Product_ID.Identifier | ValidFrom  | QtyBatch |
+      | boml_1     | bom_1                        | p_2                     | 2021-04-01 | 10       |
+    And the PP_Product_BOM identified by bom_1 is completed
+    And metasfresh contains PP_Product_Plannings
+      | Identifier | M_Product_ID.Identifier | OPT.PP_Product_BOMVersions_ID.Identifier | IsCreatePlan |
+      | ppln_1     | p_1                     | bomVersions_1                            | false        |
+
+    And metasfresh contains C_BPartners:
+      | Identifier    | Name                   | OPT.IsVendor | OPT.IsCustomer | M_PricingSystem_ID.Identifier |
+      | endcustomer_1 | EndCustomer_03032025_6 | N            | Y              | ps_1                          |
+
+    And load M_Warehouse:
+      | M_Warehouse_ID.Identifier | Value        |
+      | warehouseStd              | StdWarehouse |
+    And metasfresh contains M_HU_PI:
+      | Identifier         |
+      | huPackingLU        |
+      | huPackingTU        |
+      | huPackingVirtualPI |
+    And metasfresh contains M_HU_PI_Version:
+      | M_HU_PI_Version_ID.Identifier | M_HU_PI_ID.Identifier | Name             | HU_UnitType | IsCurrent |
+      | packingVersionLU              | huPackingLU           | packingVersionLU | LU          | Y         |
+      | packingVersionTU              | huPackingTU           | packingVersionTU | TU          | Y         |
+      | packingVersionCU              | huPackingVirtualPI    | No Packing Item  | V           | Y         |
+    And metasfresh contains M_HU_PI_Item:
+      | M_HU_PI_Item_ID.Identifier | M_HU_PI_Version_ID.Identifier | Qty | ItemType | OPT.Included_HU_PI_ID.Identifier |
+      | huPiItemLU                 | packingVersionLU              | 10  | HU       | huPackingTU                      |
+      | huPiItemTU                 | packingVersionTU              | 0   | MI       |                                  |
+    And metasfresh contains M_HU_PI_Item_Product:
+      | M_HU_PI_Item_Product_ID.Identifier | M_HU_PI_Item_ID.Identifier | M_Product_ID.Identifier | Qty | ValidFrom  |
+      | huItemPurchaseProduct              | huPiItemTU                 | p_1                     | 10  | 2020-02-01 |
+    And metasfresh contains M_Forecasts:
+      | Identifier | Name            | DatePromised | M_Warehouse_ID.Identifier |
```

Comment:
pls remove the Name column.
in StepDef, make sure you are generating a unique name... see de.metas.cucumber.stepdefs.DataTableRow#suggestValueAndName

Reply:
remove ".Identifier" from column. IMHO it's polluting the test and the column will be resolved anyways